MaxText/llama_or_mistral_ckpt.py

```diff
@@ -129,29 +192,25 @@ def convert(base_model_path, maxtext_model_path, model_size):
   base_num_kv_heads = model_params["num_kv_heads"]
   vocab_size = model_params["vocab"]
   num_experts = model_params["num_experts"] if "num_experts" in model_params else None
+  mem_info = psutil.Process()
+  logging.debug("Memory usage: %f GB", mem_info.memory_info().rss / (1024 ** 3))
 
-  print(f"Loading the base model from {base_model_path}")
+  max_logging.log(f"Loading the base model from {base_model_path}")
   # Skip any hidden files for checkpoints
   ckpt_paths = sorted(pathlib.Path(base_model_path).glob("[!.]*.pth"))
   pytorch_vars = {}
   for i, ckpt_path in enumerate(ckpt_paths):
-    print(f"Loading checkpoint {i+1} of {len(ckpt_paths)} ...")
+    max_logging.log(f"Loading checkpoint {i+1} of {len(ckpt_paths)} ...")
     checkpoint = torch.load(ckpt_path, map_location="cpu")
     pytorch_vars[int(ckpt_path.name.split(".", maxsplit=2)[1])] = checkpoint
   pytorch_vars = [pytorch_vars[i] for i in sorted(list(pytorch_vars.keys()))]
+  # map weight names if they use HuggingFace instead of PyTorch convention
+  pytorch_vars = [_HFNamespaceMapper(var) for var in pytorch_vars]
```

Comment:
Nit: could we update this variable (with after usage) to something like `ckpt_vars` or similar to avoid confusion that vars from pytorch only?

Reply:
Good idea, I renamed them!